### ISSUE TITLE
Worse performance

### ISSUE THREAD

**Issue by Slawlight** [LiyuanLucasLiu/RAdam#7]
I have get a poor test accuracy, and I set the random seed as 1.
<img width="593" alt="屏幕快照 2019-08-17 下午1 52 48" src="https://user-images.githubusercontent.com/18410535/63207328-73468500-c0f6-11e9-8fd2-9d9f1c5e125f.png">
<img width="596" alt="屏幕快照 2019-08-17 下午1 52 40" src="https://user-images.githubusercontent.com/18410535/63207329-780b3900-c0f6-11e9-8f4e-29cd3b768d69.png">

**Comment by Slawlight** [comment]
I have also tried 5e-4 as the weight decay, the test-acc is still worse, less than 90%. I set the random seed as 1.

**Comment by LiyuanLucasLiu** [comment]
For CIFAR 10 / ImageNet, RAdam didn't outperform SGD, but in our experiments, its final test accuracy should be larger than 90.0 when the learning rate is set to 0.1 (I guess this is the learning rate you use). Can you provide a script to reproduce your results?

**Comment by Slawlight** [comment]
For RAdam the learning rate is set to 0.001. And I set beta1=0.9, beta2=0.999.

**Comment by LiyuanLucasLiu** [comment]
0.001 is too small for this task, please try 0.01 / 0.03 instead. The robustness of RAdam is not infinity, and it works well for a range of learning rates, instead of all learning rates. In image classification, the learning rate will be decayed by 0.1 two times, and the final learning rate will be too small (0.00001) in the end (if you use 0.001 at the beginning). We have some recipes at https://github.com/LiyuanLucasLiu/RAdam/blob/master/cifar_imagenet/recipes.md , you can start from there.

**Comment by Slawlight** [comment]
Got it, thank you very much.
